toolkits/spotify/arcade_spotify/tools/player.py

```diff
@@ -185,13 +185,25 @@ async def start_tracks_playback_by_id(
     ] = 0,
 ) -> Annotated[dict, "The updated playback state"]:
     """Start playback of a list of tracks (songs)"""
+
+    devices = [
```

Comment:
The case where there are no available devices exist is taken care of by `handle_404_playback_state` utility function.